h2o-docs/src/product/data-science/upliftdrf.rst

```diff
@@ -388,6 +389,15 @@ Average Excess Cumulative Uplift (AECU)
 
 The Qini value can be generalized for all AUUC metric types. So AECU for Qini metric is the same as Qini value, but the AECU can be also calculated for Gain and Lift metric type. These values are stored in ``aecu_table``.
 
+
+Variable Importance
+~~~~~~~~~~~~~~~~~~~
+
+The Variable Importance is calculated based on Uplift Gain improvement. After every split the local split improvement is calculated as difference between Uplift Gain before split and Uplift Gain after split. 
+
+The Uplift Gain is calculated based on selected ``uplift_metric``. The higher Uplift Gain the better. 
```

Comment:
Do we have some external sources that we can share here?

Reply:
Or just share the formula we are using in the code.

Reply:
The formulas are described in the ``uplift_metric`` section. :)

Reply:
IMHO we are mixing terminologies. In uplift metric is written:

`Based on the uplift_metric parameter, the distribution divergence is calculated`

IMHO uplift_metric calculates distribution divergence == uplift gain. We store the uplift gain (divergence) for each variable.

Variable importance is sum of those gains.

Am I right? If yes, than I have to look in the code, i would like to be sure just from documentation.

Is it our idea or we can share some paper on it? I remember, the link to the paper in the code saved me one time.